packages/perseus-editor/src/__tests__/editor.test.tsx

```diff
@@ -1,120 +1,142 @@
-import {ApiOptions, Dependencies, Widgets} from "@khanacademy/perseus";
+import {
+    ApiOptions,
+    Dependencies,
+    Widgets,
+    widgets,
+    Util,
+} from "@khanacademy/perseus";
 import {render, screen} from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import * as React from "react";
 import "@testing-library/jest-dom"; // Imports custom mathers
 
 import {testDependencies} from "../../../../testing/test-dependencies";
-import {wait} from "../../../../testing/wait";
-import {question1} from "../__testdata__/input-number.testdata";
 import Editor from "../editor";
+import ImageEditor from "../widgets/image-editor";
+
+import type {PropsFor} from "@khanacademy/wonder-blocks-core";
+
+const Harnessed = (props: Partial<PropsFor<typeof Editor>>) => {
+    return (
+        <Editor
+            apiOptions={ApiOptions.defaults}
+            onChange={() => {}}
+            content="[[☃ image 1]]"
+            widgets={{
+                "image 1": {
+                    type: "image",
+                    options: {
+                        backgroundImage: {
+                            url: "http://placekitten.com/200/300",
+                        },
+                    },
+                },
+            }}
+            {...props}
+        />
+    );
+};
 
 describe("Editor", () => {
+    beforeAll(() => {
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+        const ImageWidget = widgets.find((w) => w.name === "image")!;
+        expect(ImageWidget).toBeDefined();
+        Widgets.registerWidget("image", ImageWidget);
+        Widgets.registerEditors([ImageEditor]);
+    });
+
     beforeEach(() => {
         jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
             testDependencies,
         );
     });
 
-    describe("input-number widget", () => {
-        beforeEach(async () => {
-            const [InputNumberWidget, InputNumberEditor] = await Promise.all([
-                import("@khanacademy/perseus").then((m) => m.InputNumber),
-                import("../widgets/input-number-editor").then((m) => m.default),
-            ]);
-            Widgets.registerWidgets([InputNumberWidget]);
-            Widgets.registerEditors([InputNumberEditor]);
-            jest.useRealTimers();
```

Comment:
This was unnecessarily complex. All we were testing here was basic widget editor interactions.